plat/socionext/synquacer/sq_bl31_setup.c

```diff
@@ -177,3 +202,68 @@ unsigned int plat_get_syscnt_freq2(void)
 
 	return counter_base_frequency;
 }
+
+#if ENABLE_SPM && SPM_DEPRECATED
```

Comment:
I think code in this block should go to separate "sq_spm.c" file.

Reply:
ok